### PR TITLE
Compute drag process MPI output

### PR DESCRIPTION
**Pull request by rubenzorrilla**
Setting the output to be done by process 0.

### CODE REVIEW

**Review by loumalouomega — APPROVED** [review]
Everything looks OK

**Comment by rubenzorrilla** [comment]
Thanks @loumalouomega, but let's wait for either @jcotela or @RiccardoRossi  approval.

**Review by jcotela — APPROVED** [review]

**Pull request merged**
